keriati/gilded-rose-javascript
# Kristina 1 - Add test coverage for gilded_rose

# Summary
* added full code coverage for gilded rose

# Test plan
* `npm run test`
* check the coverage
* check if all of the test cases are covered properly
